default.nix

```diff
@@ -1,15 +1,24 @@
 let
   pin = builtins.fromJSON (builtins.readFile ./nixpkgs.json);
 
+  # DONTMERGE
+  #nixpkgs = builtins.fetchTarball {
+  #  url = "https://github.com/NixOS/nixpkgs-channels/archive/${pin.rev}.tar.gz";
+  #  inherit (pin) sha256;
+  #};
   nixpkgs = builtins.fetchTarball {
-    url = "https://github.com/NixOS/nixpkgs-channels/archive/${pin.rev}.tar.gz";
-    inherit (pin) sha256;
+    url = "https://github.com/nicknovitski/nixpkgs/archive/android-sdk-26-licenses.tar.gz";
```

Comment:
Can you move this fork to the expo org?

Reply:
I could, but I don't want to merge a change that points our pin at a fork.  That would effectively mean maintaining our own nixpkgs channel, which I do not want.  Instead I'm going to only merge this PR when the upstream changes it depends on get merged.

Reply:
Phew, they have been merged.